corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -266,6 +275,98 @@ public int dropTable(String namespace, String tablename) {
         return tableSize;
     }
 
+    public int editRecord(String namespace, String tableName,
+        String keyToEdit, String newRecord) {
+        verifyNamespaceAndTablename(namespace, tableName);
+
+        if (keyToEdit == null) {
+            throw new IllegalArgumentException("Please specify the key to " +
+                "edit --keyToEdit");
+        }
+
+        if (newRecord == null) {
+            throw new IllegalArgumentException("Please specify the newRecord " +
+                "--newRecord");
+        }
```

Comment:
These validity checks don't look correct. What if it's not a null, but an empty string ?

Reply:
if it is an empty string, the parser will throw an InvalidProtocolBufferException.  I can add a test for this and other corner cases if can think of something additional.

Reply:
Looks like you catch the InvalidProtocolBufferException and return a null, then the editRecord will return 0, which means it will fail silently. Right?

Reply:
the new behavior now has a void return type.  It will only log errors.  Not throwing exceptions because error message is more useful on the CLI.